### PR TITLE
Add correction and amendments link to contracting phase

### LINKED ISSUE #2917: Contractor can request changes to the contract
## Additional context
This is a sub-issue of 

## User story

> As a contractor, I need to  download, sign and then upload a signed contract. 

Option 1: "contractor has no amendments" route: current manual (download, read, sign, upload) workflow 

### 
image

### 
image

###
image

###
image

Option 2: "contractor has 'simple' amendments" route relies on it being clear to the contractor they are required to communicate via the communications tab. 

## Priority
- Low

## Ideal deadline
December 2022

### PR DESCRIPTION
**Pull request by sandeepsajan0**
Fixes #2917

Add description here.

### CODE REVIEW

File: hypha/apply/projects/templates/application_projects/project_detail.html [hunk 1]
```diff
@@ -214,6 +214,13 @@ <h5>{% trans "Contracts" %}</h5>
                             - {% trans "Approved by" %} {{ contract.approver }}
                     </p>
                     {% endfor %}
+
+                    {% block correction_in_contract %}
+                        <a class="link link--bold" href="#communications" target="_blank">
+                            <svg class="icon icon--info-circle"><use xlink:href="#info-circle-fill"></use></svg>
+                                <u>Corrections or amendments to your contract?</u>
```
Comment (frjo) [inline hypha/apply/projects/templates/application_projects/project_detail.html]: I think we should use "span" and not "u" tag here.

**Review by frjo — CHANGES_REQUESTED** [review]

**Comment by theskumar** [comment]
@sandeepsajan0 would be possible for you to upload a screen for the UI changed here?

**Comment by sandeepsajan0** [comment]
@theskumar It contains the changes for the link below the Approve Contract button. 

image

File: hypha/apply/projects/templates/application_projects/project_detail.html (hunk 1 above)
Comment (sandeepsajan0) [reply hypha/apply/projects/templates/application_projects/project_detail.html]: @frjo I used "u" just to underline the link as per the design. If underlining is not necessary then yes "span" would be a better option.

**Review by frjo — APPROVED** [review]

**Pull request merged**